src/pt/portugamangas/src/eu/kanade/tachiyomi/extension/pt/portugamangas/PortugaMangas.kt

```diff
@@ -46,10 +67,10 @@ class PortugaMangas : ParsedHttpSource() {
     override fun popularMangaNextPageSelector(): String? = null
 
     override fun latestUpdatesRequest(page: Int): Request {
-        val url = "$baseUrl".toHttpUrl().newBuilder()
+        val url = baseUrl.toHttpUrl().newBuilder()
             .addQueryParameter("pagina", "$page")
             .build()
-        return GET(url, headers)
+        return client.newCall(GET(url, headers)).request()
```

Comment:
What is this for? Why not just return the request?

Reply:
There's really no need since  [fetchPopularManga(page: Int): Observable<MangasPage>](https://github.com/mihonapp/mihon/blob/617bf491eee1d1010dc23c3f6df5d148700feb44/source-api/src/commonMain/kotlin/eu/kanade/tachiyomi/source/online/HttpSource.kt#L112) already configures the client with the GET request.

Reply:
Resolve: [af7c7c4](https://github.com/keiyoushi/extensions-source/pull/1486/commits/af7c7c4ebba0ca9871f73f83e2491b6f2f8b6208)